src/DIPOLE/pair_lj_cut_dipole_cut.cpp

```diff
@@ -362,7 +371,13 @@ void PairLJCutDipoleCut::init_style()
   if (!atom->q_flag || !atom->mu_flag || !atom->torque_flag)
     error->all(FLERR,"Pair dipole/cut requires atom attributes q, mu, torque");
 
+<<<<<<< HEAD
```

Comment:
Unresolved merge conflict here.
Also the internal API for requesting neighbor lists has changed, too.

Reply:
Resolved this conflict.